src/components/templates/doc-page/code-group/doc-page-code-group.view.js

```diff
@@ -19,7 +19,7 @@ export const CodeGroup = ({
   const randomKey = getRandomKey();
   return (
     <div className={styles.wrapper}>
-      {labels.length ? (
```

Comment:
FYI, @BolotskyDev. should this change handle separately?

Reply:
@ppcano I am ok with that for now